contracts/ChildChainGaugeInjectoooor.sol

```diff
@@ -142,7 +146,7 @@ contract ChildChainGaugeInjector is ConfirmedOwner, Pausable, KeeperCompatibleIn
         uint256 totalDue;
         for (uint256 idx = 0; idx < gaugeList.length; idx++) {
             Target memory target = s_targets[gaugeList[idx]];
-            totalDue = totalDue + (target.maxPeriods - target.periodNumber) * target.amountPerPeriod;
+            totalDue += (target.maxPeriods - (target.periodNumber)) * target.amountPerPeriod;
```

Comment:
I guess in the end it's to prevent people from ending up with too much balance in the vault, but not being able to easily figure it out.

It's probably not a very strong argument.